met-web/src/services/notificationService/notificationSlice.tsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { NotificationState } from './types';
+
+const initialState: NotificationState = {
+    open: false,
+    severity: 'success',
+    text: '',
+};
+
+export const notificationSlice = createSlice({
+    name: 'notification',
+    initialState,
+    reducers: {
+        openNotification: (state: NotificationState, action: PayloadAction<NotificationState>) => {
+            state.open = action.payload.open;
+            state.severity = action.payload.severity;
+            state.text = action.payload.text;
+        },
+        closeNotification: (state: NotificationState) => {
```

Comment:
I would consider passing the initial state here so everything resets after it closes. (like text goes back to empty '')

Reply:
state = initialState maybe

Reply:
We could do that but the update happens while the notification is on screen and it looks weird up to you

Reply:
Ok all good, this is fine then. I would suggest after merging this to do a bit of research on how we can make it multiple alerts. This is great, the drawback is that if we had multiple problems happening at the same time and multiple dispatches happening only one will show. We might not need that, I suppose most of the time there will be one notification showing anyway. Research on this, however, whether we can implement it or not would be helpful